internal/repositories/repositories.go

```diff
@@ -95,7 +95,7 @@ func (r *Repositories) Overlay() error {
 		if c.DstDir != "" {
 			// delete dstDir since `git worktree add` will not replace existing directories
 			if info, err := r.appFs.Stat(c.DstDir); err == nil && info.Mode().IsDir() {
-				if err := os.RemoveAll(c.DstDir); err != nil {
+				if err := r.appFs.RemoveAll(c.DstDir); err != nil {
```

Comment:
Previously, if you happened to have a directory on your system named the same thing as whatever `c.DstDir` is set to here, you won't after running Gilt's unit tests.

Whoopsie!  😆

Reply:
hahhaha yeah oopsies